src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/RazorProjectInfoHelpers.cs

```diff
@@ -130,7 +158,7 @@ private static RazorConfiguration ComputeRazorConfigurationOptions(AnalyzerConfi
         return razorConfiguration;
     }
```

Comment:
I suggested this earlier but wanted to point it out specificaly here. Consider returning a tuple rather than a return value and an out parameter.